Expensify/App
# Fix white space around top and bottom edges of extra long images

### Details

This PR fixes an issue where a white empty space was shown at the top and bottom of extra long images. 

There was a 1px border around the thumbnail image which was not taken into account while calculating the image dimensions. This resulted in the rendered image from getting squished due to box-sizing. To fix this thumbnail styles are now applied to a parent View which wraps the fixed width container that embeds the image.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5162

### Tests




### QA Steps

1. Open chat
2. Upload extra long image Please use [Sample1](https://user-images.githubusercontent.com/7823358/132546765-0e41feb9-a06f-49c4-b3ba-2d52e96f87ae.jpg) and [Sample2](https://user-images.githubusercontent.com/7823358/132547127-bd3f124b-955e-4288-b915-02a30353f10c.jpg) image
3. Once image uploaded, verify that there is no extra white space around the top and bottom edges of the extra long image.
4. Verify that existing styles for thumbnail images are applied as expected.
### Tested On


### Screenshots


#### Web

![Screenshot 2021-11-21 at 9 38 32 AM](https://user-images.githubusercontent.com/6880914/142749214-1305ee55-b0d1-4ac8-9b42-efcaa14ffee9.jpg)


#### Mobile Web

![localhost_8080_r_83562329](https://user-images.githubusercontent.com/6880914/142749232-1ec62866-ec64-432f-9251-500fee929f1f.png)


#### Desktop
![Screenshot 2021-11-21 at 9 28 58 AM](https://user-images.githubusercontent.com/6880914/142749129-5e83992e-82b1-4f30-8954-ea95ef11975a.jpg)
#### iOS
![iOS](https://user-images.githubusercontent.com/6880914/142749061-65fa7a7a-634a-45ad-998a-2fa665eccb81.jpeg)

#### Android
![Screenshot_2021-11-21-09-31-58-143_com expensify chat](https://user-images.githubusercontent.com/6880914/142749124-e4ff83c1-3ca9-4e8b-a783-285f38f404ed.jpg)
